scapy/contrib/gtp.uts

```diff
@@ -21,6 +21,7 @@ gtp = IP(src="127.0.0.1")/UDP(dport=2123)/GTPHeader(teid=2807)/GTPCreatePDPConte
 gtp.dport == 2123 and gtp.teid == 2807 and len(gtp.IE_list) == 5
 
 = GTPCreatePDPContextRequest(), basic dissection
+~ random_weird_py3
 random.seed(0x2807)
 assert raw(gtp) in [b"E\x00\x00K\x00\x01\x00\x00@\x11|\x9f\x7f\x00\x00\x01\x7f\x00\x00\x01\x08K\x08K\x007\x1c\xdb0\x10\x00'\x00\x00\n\xf7\x10A\xb77-\x14\x0f\x85\x00\x04\xd6!-b\x85\x00\x04\xbf\xf8\xc9Z\x87\x00\x0faWdWRWX0qEAXLPE",
```

Comment:
Is the list still needed ?

Reply:
Yes, because 3.6 will be different than 2.7 (random_weird_py3 is applied only for < 3.6).